modules/telco-ran-red-hat-advanced-cluster-management-rhacm.adoc

```diff
@@ -10,10 +10,10 @@ New in this release::
 * Additional node labels can be configured during installation.
 
 Description::
-{rh-rhacm} provides Multi Cluster Engine (MCE) installation and ongoing lifecycle management functionality for deployed clusters.
+{rh-rhacm} provides the {mce} installation and ongoing lifecycle management functionality for deployed clusters.
 You declaratively specify configurations and upgrades with `Policy` CRs and apply the policies to clusters with the {rh-rhacm} policy controller as managed by {cgu-operator-full}.
 +
-* {ztp-first} uses the MCE feature of {rh-rhacm}
+* {ztp-first} uses the {mce-short} feature of {rh-rhacm}
```

Comment:
Are you sure that "multicluster engine operator feature" is correct here? I would expect this to be just "multicluster engine feature", but the use of the attributes as defined adds "operator".

Reply:
Changed to "multicluster engine for Kubernetes"